projects/media-viewer/src/lib/toolbar/media-viewer-toolbar/media-viewer-toolbar.component.html

```diff
@@ -0,0 +1,193 @@
+          [class.button-hidden-on-dropdown]="isToolbarButtonVisible(mvHighlightBtn.id)"
+          [class.toggled]="toolbarEvents.highlightModeSubject | async" aria-pressed="false"
+          (click)="onClickHighlightToggle()" data-l10n-id="toggleHighlightButton">
+          <span data-l10n-id="highlight_label">Highlight</span>
+        </button>
+
+        <ng-container *ngIf="toolbarButtons.showNavigation">
+          <div id="mvPageBtn" #mvPageBtn class="mv-toolbar__menu-button--page" aria-pressed="false"
+            [class.button-hidden-on-toolbar]="!isToolbarButtonVisible(mvPageBtn.id)"
+            [class.button-hidden-on-dropdown]="isToolbarButtonVisible(mvPageBtn.id)">
+            <span>Page</span>
+            <button id="mvUpBtn" role="button" [disabled]="pageNumber === 1" title="Previous Page"
+              class="mv-toolbar__menu-button--up button-image" data-l10n-id="previous"
+              (click)="decreasePageNumber()"><span></span></button>
+            <button id="mvDownBtn" role="button" [disabled]="pageNumber === pageCount" title="Next Page"
+              class="mv-toolbar__menu-button--down button-image" data-l10n-id="next"
+              (click)="increasePageNumber()"><span></span></button>
+
+            <input type="string" appInteger id="pageNumber" class="hmcts-toolbar-input govuk-input--width-2"
+              title="Page Number" value="1" size="4" min="1" [value]="pageNumber" aria-label="page number" tabindex="0"
+              data-l10n-id="page" (change)="onPageNumberInputChange(pageNumberInput.value)" #pageNumberInput>
+            <span id="numPages" class="toolbarLabel">/ {{ pageCount }}</span>
+          </div>
+        </ng-container>
+
+        <ng-container *ngIf="toolbarButtons.showZoom">
+          <div id="mvZoomBtn" #mvZoomBtn class="mv-toolbar__menu-button--zoom" aria-pressed="false"
+            [class.button-hidden-on-toolbar]="!isToolbarButtonVisible(mvZoomBtn.id)"
+            [class.button-hidden-on-dropdown]="isToolbarButtonVisible(mvZoomBtn.id)">
+            <span>Zoom</span>
+            <button [disabled]="toolbarEvents.zoomValueSubject.value == 0.1" id="mvMinusBtn"
+              class="mv-toolbar__menu-button--zoom-out button-image" title="Zoom Out" data-l10n-id="zoom_out"
+              (click)="stepZoom(-0.1)"><span></span></button>
+            <select id="scaleSelect" class="hmcts-toolbar-select" title="Zoom" tabindex="0" data-l10n-id="zoom"
+              (change)="zoom($event.target.value)" aria-label="zoom">
+              <option #zoomSelect id="customScaleOption" title="" [value]="toolbarEvents.zoomValueSubject.value">
+                {{(toolbarEvents.zoomValueSubject.value) * 100 | number: '1.0-0'}}%</option>
+              <option title="" value="0.1" data-l10n-id="page_scale_percent" data-l10n-args='{ "scale": 10 }'>10%
+              </option>
+              <option title="" value="0.25" data-l10n-id="page_scale_percent" data-l10n-args='{ "scale": 25 }'>25%
```

Comment:
should this be a for loop?

Reply:
That's how it was originally, however I have put it in an ngFor loop and set the values from the ts file.